docs/DeveloperGuide.md

```diff
@@ -1023,10 +1023,14 @@ Priorities:
 | `* * *`  | new user                                   | see a list of available commands | refer to instructions when I forget how to use the application       |
 | `* * *`  | user                                       | create a budget                | add a new budget that I need                                           |
 | `* * *`  | user                                       | delete a budget                | remove a budget that I no longer need                                  |
+| `* * *`  | user                                       | edit a budget                  | edit the name or the threshold that needs to be changed                |
+| `* * `   | user                                       | sort my budgets                | view my budgets based on name or date                                  |
 | `* * *`  | user                                       | open a budget                  | view the expenditures of a budget                                      |
 | `* * *`  | user                                       | close a budget                 | return to the main window                                              |
 | `* * *`  | user                                       | add an expenditure             | add a new expenditure to a budget                                      |
 | `* * *`  | user                                       | delete an expenditure          | remove an expenditure that I no longer need                            |
+| `* * *`  | user                                       | edit an expenditure            | edit the name, price and tag that needs to be changed                  |
+| `* * `   | user                                       | sort my expenditures           | view my expenditures based on name or date                             |
 | `* *`    | user                                       | find an expenditure by name    | locate an expenditure easily                                           |
```

Comment:
change 'by name' to 'by search term'

Reply:
add clear budget user story, add find budget/expenditure user story, list budget/expenditure user stories 👍

Reply:
Add user story for indication if expenditure exceeds threshold

Reply:
Add user story to see how many expenditures inside a budget

Reply:
Add user story to include tags to my expenditures

Reply:
Add user story for exiting the program,for seeing latest entries at top of budgets, for seeing latest entries at top of expenditures, when an expenditure is created

Add user story for new user to look through sample data